Source/Core/DolphinWX/UINeedsControllerState.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <atomic>
+
+static std::atomic<bool> s_needs_controller_state{false};
```

Comment:
I think std::atomic is probably appropriate here, it's not really using flag functionality (the compare-and-swap stuff).